File: online-judge-solutions/Codeforces/1675A.py
#https://codeforces.com/problemset/problem/1675/A
#Lang		:	Python 3.8
#Time		:	124 ms
#Memory         :	0 KB
for _ in range(int(input())):
    a,b,c,x,y=map(int,input().split())
    print("YES" if (((a-x)*(a<x)+(b-y)*(b<y)+c)>=0) else "NO")

File: online-judge-solutions/Codeforces/1675A.cpp
//https://codeforces.com/problemset/problem/1675/A
//Lang		:	GNU C++14
//Time		:	30 ms
//Memory	:	0 KB
#include<iostream>
using namespace std;
#define fastInp cin.tie(0); cout.tie(0); ios_base::sync_with_stdio(0);
int main(){
	fastInp;
	int t,a,b,c,x,y;
	cin>>t;
	while(t--){
		cin>>a>>b>>c>>x>>y;
		cout<<((((a-x)*(a<x)+(b-y)*(b<y)+c)>=0)?"YES":"NO")<<"\n";
	}
	return 0;
}
